bindsnet/utils.py

```diff
@@ -183,3 +183,44 @@ def reshape_locally_connected_weights(
                             ]
 
         return square
+
+
+def reshape_conv2d_weights(
+    weights: torch.Tensor,
+    wmin: float = 0.0,
```

Comment:
I moved the clamping functionality away from `reshape_conv2d_weights`. I think it makes more sense to do this at the plotting level.

Reply:
Actually, should clamping be done at either level? Shouldn't it only be done in the network code?

Reply:
That's a fair argument. I guess since it was being done by the `plot_conv2d_weights` in `plotting.py`, I was considering keeping it. It might make more sense to normalize then plot.